framework/src/predictors/SimplePredictor.C

```diff
@@ -41,14 +41,7 @@ SimplePredictor::shouldApply()
 void
 SimplePredictor::apply(NumericVector<Number> & sln)
 {
-  // Save the original stream flags
-  std::ios_base::fmtflags out_flags = Moose::out.flags();
-
-  _console << "  Applying predictor with scale factor = " << std::fixed << std::setprecision(2)
```

Comment:
Another option is to use `Moose::out` here instead of _console. This stream gets reset by `Moose::out.flags(out_flags)` below. However, I wasn't sure i f there is a real reason to use `_console` instead of `Moose::out`?

Reply:
`_console` respects the options given by the user (say if they want to turn it off, or have it print less frequently). More importantly it gets indented when used in a Multiapp. `Moose::out` is just `std::cout`. You don't want to use `Moose::out` for anything but temporary debugging.

Reply:
Good to know. Then the current implementation makes the most sense. There are probably ways to change the output format temporarily, but I don't think it's necessary and will hide values (i.e. it'll show 0 if predictor is less than 0.01)

Reply:
This "fix" may or may not be good permantely. As we refactor things around you might run into new problems where the precision or output type may not be what you want because another system is setting and not resetting flags properly. 

What probably should be done is we have a standard that we agree on (say scientific with some specific precision), and if you are ok with that you should have to worry about saving and resetting flags. If however somebody wants something different they should restore the flags to the default before that routine exits. Seems like a good intern project!

Reply:
Yeah, I agree. This fix was due to SimplePredictor screwing up the flags for reference residual norms.